polaris-intellisense/src/Polaris/PolarisAutocomplete.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+import {allTokens} from '../data/allTokens';
+
+export class PolarisAutocomplete implements vscode.CompletionItemProvider {
+  get allTokens() {
+    return allTokens;
+  }
```

Comment:
Does this need to be a getter?
```suggestion
  allTokens = allTokens;
```

Reply:
I'm actually not seeing this instance prop used anywhere. Is it needed?